settings/distros.go

```diff
@@ -0,0 +1,37 @@
+package settings
+
+type PackageManager string
+
+const (
+	Apt = "apt"
+	Yay = "yay"
+	Dnf = "dnf"
+	Apk = "apk"
+)
+
+func (p PackageManager) ToString() string {
```

Comment:
This function is also never used. In Go you can do `string(distroInfo.Pkgmanager)` to convert the value to a string.